scripts/Controller.ice

```diff
@@ -8,6 +8,7 @@ sequence<string> StringSeq;
 
 class Config
 {
+    ["swift:identifier:transportProtocol"]
```

Comment:
Is all this code Python-only? I'd rather avoid confusing metadata that suggests it's used in Swift.

Reply:
The controller is implemented by iOS and Android test controller applications. With iOS we have one for C++ and one for Swift. And there is a Java implementation for android.

Reply:
Indeed, I forgot to update the Swift iOS test code. But I've updated it now.

Even though it looks like iOS just completely ignores this stuff, we do still compile the generated code, which means that we do need to remap these identifiers.